code/modules/goonchat/browserassets/js/browserOutput.js

```diff
@@ -35,7 +35,7 @@ var opts = {
 	'wasd': false, //Is the user in wasd mode?
 	'priorChatHeight': 0, //Thing for height-resizing detection
 	'restarting': false, //Is the round restarting?
-	'darkmode':true, //Are we using darkmode? If not WHY ARE YOU LIVING IN 2009???
+	'darkmode':false, //Are we using darkmode? If not WHY ARE YOU LIVING IN 2009???
```

Comment:
People kept REEEing at me because it was default

Reply:
I'm sure part of that is that the dark mode was pretty bad and actually had some major flaws.
If it wasn't then they can cry and piss their pants and shit and cum some more because it's literally one button click to fix.

Reply:
revert